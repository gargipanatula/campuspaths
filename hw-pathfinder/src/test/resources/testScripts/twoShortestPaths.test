# Find a path when there are two equal ones available

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C
AddNode simpleGraph D
AddNode simpleGraph E
AddNode simpleGraph F

AddEdge simpleGraph A B 1.0
AddEdge simpleGraph A D 1.0
AddEdge simpleGraph B C 2.0
AddEdge simpleGraph D E 2.0
AddEdge simpleGraph C F 1.0
AddEdge simpleGraph E F 1.0

FindPath simpleGraph A F
