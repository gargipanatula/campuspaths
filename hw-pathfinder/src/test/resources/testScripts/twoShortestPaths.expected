# Find a path when there are two equal ones available

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph
added node D to simpleGraph
added node E to simpleGraph
added node F to simpleGraph

added edge 1.000 from A to B in simpleGraph
added edge 1.000 from A to D in simpleGraph
added edge 2.000 from B to C in simpleGraph
added edge 2.000 from D to E in simpleGraph
added edge 1.000 from C to F in simpleGraph
added edge 1.000 from E to F in simpleGraph

path from A to F:
A to B with weight 1.000
B to C with weight 2.000
C to F with weight 1.000
total cost: 4.000